packages/github_backup.py

```diff
@@ -103,6 +104,7 @@ def get_repos(org, repo_type, access_token=None, username=None, password=None, p
             add_https_basic_auth(request, username, password)
         response = urlopen(request)
         pagination = get_pagination(response.headers['Link'])
+        print(repr(pagination))
```

Comment:
did you mean to leave these print statements in?

Reply:
I did not!